apps/admin-portal/src/components/roles/create-role-wizard/role-permisson.tsx

```diff
@@ -17,12 +17,12 @@
  */
 
 import React, { FunctionComponent, ReactElement, useEffect, useState} from "react";
-import { getPermissionList, getPermissionsPerRole } from "../../../api";
+import { getPermissionList, getPermissionsForRole } from "../../../api";
 import { Permission } from "../../../models/permission";
 import { TreeView } from "@wso2is/react-components";
 import { Forms } from "@wso2is/forms";
 import { addPath } from "../role-utils";
-import { Segment, Button, Grid } from "semantic-ui-react";
+import { Segment, Button, Grid, Loader } from "semantic-ui-react";
```

Comment:
We can use the `ContentLoader` our react component library.

Reply:
FIxed with commit 93daf77